src/service/resource.js

```diff
@@ -556,9 +556,9 @@ export class Resource {
    * @return {boolean}
    */
   isDisplayed() {
-    return (this.layoutBox_.height > 0 && this.layoutBox_.width > 0 &&
+    return (this.layoutBox_.height > 0 || this.layoutBox_.width > 0) &&
```

Comment:
Why && -> ||? Seems a bit unintuitive, so if correct, a comment would be beneficial.

Reply:
I'd rather keep the old rule, but enable || for fluid only.

Reply:
Fixed. The function accepts either a positive height, or a height of 'fluid'.